src/summary/MedicationRangeChartVisualizer.jsx

```diff
@@ -74,10 +74,11 @@ class MedicationRangeChartVisualizer extends Component {
         return rows;
     }
 
-    renderMedicationTitle = (lowerValue, upperValue, name, dosageValue, dosageUnit, timingValue, timingUnit) => {
+    renderMedicationTitle = (lowerValue, upperValue, name, dosageValue, dosageUnit, timingValue, timingUnit, asNeededIndicator) => {
         // Determining if timingUnit has a value. Set if empty string if null.
         if (timingUnit == null) timingUnit = '';
-
+        //Determining if asNeededIndicator is true. Set empty string if false, set string as needed if true.
+        asNeededIndicator ? asNeededIndicator = 'as needed' : asNeededIndicator = '';
```

Comment:
[DP1] We probably shouldn't be changing argument values here as a best practice. I know it's just a string here, but it's probably not the best thing for us to have code-quality wise. Any chance we can just have a local variable here instead? Something like `const asNeededString = (asNeededIndicator ? "as needed" : "");`